core/chains/evm/gas/rollups/op_l1_oracle.go

```diff
@@ -92,94 +94,51 @@ const (
 	ScrollGasOracleAddress = "0x5300000000000000000000000000000000000002"
```

Comment:
With the new changes, think we can clear out these unused addresses above: `OPGasOracleAddress`, `KromaGasOracleAddress`, and `ScrollGasOracleAddress`